test/runner/provider/test.js

```diff
@@ -100,6 +101,33 @@ define([
             .init();
     });
 
+    QUnit.test('extend default API with APIP data', function(assert) {
```

Comment:
This is not a test **with** APIP data.
This is a test **without**.

A correct wording could be `Checking APIP without data`